politeiawww/invoices.go

```diff
@@ -681,6 +659,27 @@ func (p *politeiawww) validateInvoice(ni cms.NewInvoice, u *user.User) error {
 					}
 				}
 			}
+
```

Comment:
Are you suggesting using another version of validateInvoice for processEditInvoice?

Reply:
I was suggesting moving this logic to `processNewInvoice` and `processEditInvoice` instead of having an `isEdit` arg.

So in `processNewInvoice` you would check to make sure that no other invoices had the payment address.  In `processEditInvoice` you would check to make sure that only one invoice has the same payment address and it was the invoice being edited.